tests/acceptance/lib/files/replace_uncommented_substrings.cf

```diff
@@ -0,0 +1,41 @@
+body common control
+{
+        inputs => { '../../default.cf.sub',
+        };
+        bundlesequence  => { default("$(this.promise_filename)") };
+        version => "1.0";
+}
+#######################################################
+
+bundle common classes
+{
+  classes:
+
+      "testing_masterfiles_policy_framework"
```

Comment:
Did we log a ticket to clean this up? I would assume this is the case in all masterfiles tests/acceptance tests.

Reply:
https://tracker.mender.io/browse/ENT-6132